app/controllers/vouchers_controller.rb

```diff
@@ -83,18 +83,22 @@ def create
     if shouldEmail
       email_confirmation(:confirm_reservation, @customer, theshowdate, thenumtoadd)
     end
-    
+
```

Comment:
what is this line for?

Reply:
just an extra whitespace, dont want to make a commit just to fix it, it doesnt change anything